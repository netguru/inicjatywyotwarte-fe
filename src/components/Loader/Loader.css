.loader {
  text-align: center;
  margin: 4em;
}
