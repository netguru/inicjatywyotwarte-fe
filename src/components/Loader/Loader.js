import React from 'react'
import styled from 'styled-components'

const StyledLoader = styled.div`
  text-align: center;
  margin: 4em;
`
StyledLoader.displayName = 'StyledLoader';

export default function Loader () {
  return <StyledLoader>≈Åadowanie...</ StyledLoader>
}
