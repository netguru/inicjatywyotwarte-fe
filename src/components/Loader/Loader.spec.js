import React from 'react'
import Loader from './Loader'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Loader', () => {
  test('renders loader', () => {
    const wrapper = shallow(<Loader/>);
    expect(wrapper.find('StyledLoader').length).toEqual(1);
    expect(wrapper.find('StyledLoader').text()).toEqual('Ładowanie...');
  });


  test('match snapshot', () => {
    const wrapper = shallow(<Loader/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});


