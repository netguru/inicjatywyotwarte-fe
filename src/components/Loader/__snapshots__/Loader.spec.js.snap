// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader match snapshot 1`] = `
<StyledLoader>
  ≈Åadowanie...
</StyledLoader>
`;
